Openshift-EE/pipelines/ElasticSearch/stages/2-setup/12PS-storage-policies/openebs-sc

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+set -x
+pod() {
+sshpass -p $pass ssh -o StrictHostKeyChecking=no $user@$ip -p $port 'git clone -b OpenEBS-base https://github.com/openebs/e2e-openshift.git'
```

Comment:
@sushma1118 this test scripts is for elasticsearch pipeline. Clone the elastic search branch.